test/PizzeriaTest.cpp

```diff
@@ -65,3 +71,35 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTestMain, shouldDuplicateMain)
+{
+    constexpr minutes funghiBakingTime = minutes(4);
+    constexpr minutes margheritaBakingTime = minutes(3);
+
+    constexpr double cheaperPizza = 10.99;
+    constexpr double moreExpensivePizza = 250.0;
+
+    NiceMock<PizzaMock> niceMockPizza{};
+    StrictMock<PizzaMock> strictMockPizza{};
+
+    Pizzeria bravo("Bravo Pizza", mockTimer);
+    Pizzas pizzas = {new PizzaStub("Stub"), new PizzaStub("Stub2"), new PizzaStub("Stub3"), &niceMockPizza, &strictMockPizza};
+
+    EXPECT_CALL(mockTimer, sleep_for(minutes(1))).Times(3);
```

Comment:
avoid magic values for time :)

Reply:
+1

Reply:
just tests :)

Reply:
Yup, avoid magic values in tests as well. What is minutes(1)? You should rather have variable
`minutes stubBakingTime = minutes(1);` and the test will be more readable.